packages/lib/server/repository/event.ts

```diff
@@ -1,18 +0,0 @@
-import { getPublicEvent } from "@calcom/features/eventtypes/lib/getPublicEvent";
-import prisma from "@calcom/prisma";
-import type { TEventInputSchema } from "@calcom/trpc/server/routers/publicViewer/event.schema";
-
-export class EventRepository {
```

Comment:
let's remove this repository since we can just use `getPublicEvent` directly from features

Reply:
I think it's the other way around. We should keep repositories as it is the preferred pattern over single use helper functions.

Reply:
gottcha